### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/cookbook/vuepress/theme.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#694] [Announcements]
# https://theme-hope.vuejs.press/zh/cookbook/vuepress/theme.html

VuePress 主题为你提供了布局、样式和其他功能，帮助你专注于 Markdown 内容的写作。 默认主题 VuePress 有一个开箱即用的默认主题，正使用在你当前正在浏览的文档网站上。 如果你不指定要使用的主题，那么就会自动使用默认主题。 为了配置默认主题，你需要在你的配置文件中通过 theme (https://vuejs.press/zh/r...

https://theme-hope.vuejs.press/zh/cookbook/vuepress/theme.html

**Comment by xzy111222333** [comment]
讲解的太全了，谢谢
